views/index.pug

```diff
@@ -37,6 +38,10 @@ html
                 a.nav-link(onclick="load_view('receive')")
                   span.fas.fa-piggy-bank
                   span &nbsp; Receive
+              li.nav-item
+                a.nav-link(onclick="load_view('walletConfig')")
+                  span.fas.fa-tablet(data-fa-transform="rotate--90")
```

Comment:
There's an `fa-wallet` icon, which probably makes the most sense here. The `adjust balances` tab added by DJ uses it already, but you could change that to something like `fa-balance-scale`